samples/force-sensitive-resistor/FsrWithAdcSample.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Iot.Device.Mcp3008;
+
+namespace force_sensitive_resistor
```

Comment:
ForceSensitiveResistor

Reply:
I am actually followed naming structure of existing samples https://github.com/dotnet/iot/blob/master/samples/led-blink/Program.cs#L9